### DISCUSSION TITLE
Feature Request: Allow setting icon color per device to know active audio device at a glance (for those who add or remove multiple audio outputs)

### DISCUSSION THREAD

**Discussion by NemukaPrime** [File-New-Project/EarTrumpet#1589] [Ideas]
**Description:**
Hello, I want to be able to know at a glance what my active audio device is. I think that can be done by having the option to change the color of the volume icon per device:
A user can have red for 3.5mm headphone jack, blue for HDMI monitor speakers, green for Bluetooth headphones, etc. All user configurable of course, so that people who want the default icon can stay with the old icon.

**Problem:**
Sometimes, I try to increase or decrease the volume through mouse scroll, but it ends up doing nothing when I connect a Bluetooth device. This isn't really an EarTrumpet problem, but the Windows behavior not being consistent in my case: sometimes, it switches to the BT device and sometimes it stays with the previous device (3.5mm headphone jack output).

It kind of makes scroll volume control useless for me since I have to right click to know the current device, switch to the new device if necessary, then click outside to remove the popup, and THEN left click to open the volume mixer panel so I can change the volume.

I think the option to change the colors would be helpful for those with multiple audio outputs who constantly switch between them. It can also reduce the number of clicks needed to control volume since confirmation wouldn't be required anymore. 

I hope this or a similar solution can be implemented. Thank you.

**Comment by riverar** [comment]
Would adding the device name to the tooltip be helpful?

**Reply by NemukaPrime** [reply]
If the tooltip is the yellow box with text that appears when I hover the cursor over the icon, then I think it already shows the device name in the tooltip.

I'd still rather know (through color or some other way) by just looking at it without needing to hover over and wait, since going to the icon and clicking it is faster for me.

An example is ClickMonitorDDC, which has system tray icons that can be mouse scrolled to change different values per monitor for contrast, brightness, saturation, etc.

![ClickMonitorDDC example ](https://github.com/user-attachments/assets/29373ae6-74fa-40ff-a792-a7778c9b85bf)

The contrast and brightness icons are different colors. And in the case of more than one monitor, their individual icon fonts can be changed so you know just by looking which monitor's controls you're changing.
